website/docs/sdk-reference/php.md

```diff
@@ -110,12 +119,129 @@ $client = new \ConfigCat\ConfigCatClient("#YOUR-SDK-KEY#");
 $keys = $client->getAllKeys();
 ```
 
+## Flag Overrides
+
+With flag overrides you can overwrite the feature flag & setting configuration fetched from the ConfigCat CDN with local values.
```

Comment:
```suggestion
With flag overrides you can overwrite the feature flag & setting configuration downloaded from the ConfigCat CDN with local values. This could be useful when ... 
```